src/models/todos.ts

```diff
@@ -0,0 +1,19 @@
+import mongoose, { Schema, Document } from 'mongoose';
+
+export interface ITodo extends Document {
+  title: string;
+  userId: mongoose.Types.ObjectId;
+  isCompleted: boolean;
+  createdDate: Date;
+  updatedDate: Date;
+}
+
+const TodoSchema: Schema = new Schema({
+  title: { type: String, required: true },
+  userId: { type: mongoose.Types.ObjectId, ref: 'User', required: true },
+  isCompleted: { type: Boolean, required: true, default: false },
+  createdDate: { type: Date, default: Date.now },
```

Comment:
Instead of defining createdDate and updatedDate Use it like below it will handle dates automatically.

timestamps: { createdAt: 'createdDate', updatedAt: 'updatedDate' }

Reply:
Done